.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.3.0
+    rev: v3.2.0
```

Comment:
I'm not sure why https://github.com/googleapis/synthtool/pull/866 isn't in effect here (merged 2020-12-02).